roles/lib_utils/tasks/main.yml

```diff
@@ -8,4 +8,6 @@
   package:
     name: python-ruamel-yaml
```

Comment:
I think it'll probably be a good idea to conditionalize python2-ruamel-yaml/python3-ruamel-yaml depending on not-Fedora/Fedora